torch/distributions/transforms.py

```diff
@@ -61,12 +61,16 @@ class Transform(object):
             the codomain. Transforms that are not bijective should at least
             maintain the weaker pseudoinverse properties
             ``t(t.inv(t(x)) == t(x)`` and ``t.inv(t(t.inv(y))) == t.inv(y)``.
+        sign (int or Variable): For bijective univariate transforms, this
+            should be +1 or -1 depending on whether transform is monotone
+            increasing or decreasing.
         event_dim (int): Number of dimensions that are correlated together in
             the transform ``event_shape``. This should be 0 for pointwise
             transforms, 1 for transforms that act jointly on vectors, 2 for
             transforms that act jointly on matrices, etc.
     """
     bijective = False
+    sign = float('nan')
```

Comment:
I think it would be a better idea to make it a property that raises `NotImplementedError` instead of a `NaN`